analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -627,30 +627,27 @@ def add_arguments_to_parser(parser):
         """
 Checkers
 ------------------------------------------------
-The analyzer performs checks that are categorized into families or "checkers".
-See 'CodeChecker checkers' for the list of available checkers. You can
-fine-tune which checkers to use in the analysis by setting the enabled and
-disabled flags starting from the bigger groups and going inwards, e.g.
-'-e core -d core.uninitialized -e core.uninitialized.Assign' will enable every
-'core' checker, but only 'core.uninitialized.Assign' from the
-'core.uninitialized' group. Please consult the manual for details. Disabling
-certain checkers - such as the 'core' group - is unsupported by the LLVM/Clang
-community, and thus discouraged.
+The analyzer checks the source code with the help of checkers. Checkers
+implement a specific rule, such as "don't divide by zero", and emit a warning
+if said rule is violated. You can see the list of available checkers with
+'CodeChecker checkers'.
+
+Checkers are assigned to several groups based several variables. While
+individual analyzers might create their own groups, such as 'clangsa' has so
+called packages, in addition, CodeChecker orders 'labels' and 'guidelines' to
+checkers, which are detailed below.
```

Comment:
It should be described simply that the grouping of checkers can either be defined by the analyzer itself, or through labels. Guideline and profile are such labels. It may be deceiving to write "labels and guidelines".

As for checker groups, clang is a little special one, because it defines its groups along the delimiter characters in the checker names. But cppcheck has custom group names.